.github/workflows/test.yml

```
@@ -126,6 +126,8 @@ jobs:
       - uses: Swatinem/rust-cache@v1
       - name: Run
         run: cargo check-all-features
+      - name: Bench Check
+        run: cargo bench --no-run --features full
```

```suggestion
        run: cargo bench --no-run --features full,benchmarks
```

sorry, my bad: we also need the benchmarks here